templates/participant/details.html.twig

```diff
@@ -2,6 +2,7 @@
 {% block title %}Participant {{ participant.id }} - {% endblock %}
 {% block body %}
     {% import 'macros/display-text.html.twig' as macros %}
+    <div id="participant_info" data-participant-id="{{ participant.id }}" data-org-patient-status="{{ orgPatientStatusData|json_encode }}" data-can-view-patient-status="{{ canViewPatientStatus }}">
```

Comment:
It looks like the `orgPatientStatusData` is only used to determine whether there are any patient status data or not. Instead of passing in the entire structure, could you just determine here whether or not it's empty and then set it to false/true or 0/1 or something like that?

Reply:
Alternatively, if you're going to keep the entire structure here, I think you might need to change the escape strategy to: `orgPatientStatusData|json_encode|escape('html_attr')` since it's inside an attribute.